src/engine/scenes/TestSceneKevin.cpp

```diff
@@ -68,78 +68,24 @@ namespace lei3d {
     }
 
     void TestSceneKevin::OnUpdate(float deltaTime) {
+        GLFWwindow* const window = Application::Curr()->Window();
+        
 
+        if (glfwGetKey(window, GLFW_KEY_R) == GLFW_PRESS)
+        {
+            Entity* backpackEntity = GetEntity("Backpack");
+            if (backpackEntity)
+            {
+                std::cout << "backpackEntity" << std::endl;
+                backpackEntity->SetPosition(glm::vec3(0.0f, 0.0f, 0.0f)); // IS THIS OKAY? DOES IT CAUSE A MEMORY LEAK?
```

Comment:
I don't think so since you're not creating any new memory. If it does that's an Entity class problem

Reply:
yeah I'm tripping.